client/src/app/pages/migration-targets/components/custom-target-form.tsx

```diff
@@ -105,6 +117,7 @@ export const CustomTargetForm: React.FC<CustomTargetFormProps> = ({
     .object()
     .shape({
       id: yup.number().defined(),
+      provider: yup.string().defined(),
```

Comment:
More informational... I'm pretty sure something like this would work:
```suggestion
      provider: yup.string().oneOf(["Java", "Go"]).defined(),
```